applications/asset_tracker/README.rst

```diff
@@ -52,6 +52,49 @@ The `LTE Link Monitor`_ application, implemented as part of `nRF Connect for Des
 
 By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
 
+.. _power_opt:
+
+Power optimization
+==================
+
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+These settings are currently only supported on the nRF9160 DK.
+
+.. note::
+   Not all cellular network providers support these modes, and the granted parameters can vary between networks.
+
+Demo mode
+	This is the default setting.
+	In this mode, the device maintains a continuous cellular link.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=n``.
+
+Request eDRX mode
+	In this mode, the device requests the eDRX feature from the cellular network to save power.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
+	set Switch 2 to the N.C. position.
+
+Request Power Saving Mode (PSM)
+	In this mode, the device requests the PSM feature from the cellular network to save power.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
+	set Switch 2 to the GND position.
+
+Concurrent GPS and LTE
+++++++++++++++++++++++
+
+An nRF9160-based device supports LTE-M and NB-IoT along with power saving features such as eDRX and PSM.
+The following figure shows how the data transfer occurs in an nRF9160-based device with energy conservation in place.
```

Comment:
GPS will also work without these two modes, but it's very case-dependant if that works well. Generally, we would very much encourage the use of eDRX and/or PSM (both can be used at the same time) when using the GPS.

Reply:
Added the information:  Concurrent operation of GPS with optional power saving features such as extended Discontinuous Reception (eDRX) and Power Saving Mode (PSM) is also supported and recommended.